docs-en/chapter_preface/suggestions.md

```diff
@@ -2,223 +2,225 @@
 
-![Clone repository with download code](suggestions.assets/download_code.png)
+![Cloning Repository and Downloading Code](suggestions.assets/download_code.png)
 
-**Step 3: Run the source code**. As shown in the figure below, for the code block labeled with the file name at the top, we can find the corresponding source code file in the `codes` folder of the repository. The source code files can be run with a single click, which will help you save unnecessary debugging time and allow you to focus on what you are learning.
+**Step 3: Run the source code**. As shown in the figure below, for the code block labeled with the file name at the top, we can find the corresponding source code file in the `codes` folder of the repository. These files can be executed with a single click, which will help you save unnecessary debugging time and allow you to focus on learning.
 
-![Code block with corresponding source file](suggestions.assets/code_md_to_repo.png)
+![Code Block and Corresponding Source Code File](suggestions.assets/code_md_to_repo.png)
 
-## Growing Together In Questioning And Discussion
+## Learning Together in Discussion
 
-While reading this book, please don't skip over the points that you didn't learn. **Feel free to ask your questions in the comment section**. We will be happy to answer them and can usually respond within two days.
+While reading this book, please don't skip over the points that you didn't learn. **Feel free to post your questions in the comment section**. We will be happy to answer them and can usually respond within two days.
 
-As you can see in the figure below, each post comes with a comment section at the bottom. I hope you'll pay more attention to the comments section. On the one hand, you can learn about the problems that people encounter, so as to check the gaps and stimulate deeper thinking. On the other hand, we expect you to generously answer other partners' questions, share your insights, and help others improve.
+As you can see in the figure below, each chapter has a comment section at the bottom. I hope you'll pay more attention to the comments section. On the one hand, you can learn about the problems others have encountered, which can help you identify gaps in your knowledge and inspire deeper thinking. On the other hand, we expect you to generously answer other readers' questions, share your insights, and help others improve.
 
-![Example of comment section](../index.assets/comment.gif)
+![Comment Section Example](../index.assets/comment.gif)
 
-## Algorithm Learning Route
+## Algorithm Learning Path
 
-From a general point of view, we can divide the process of learning data structures and algorithms into three stages.
+Overall, we can divide the process of learning data structures and algorithms into three stages.
```

Comment:
"Overall, we can divide the process of learning data structures and algorithms into three stages."
can be changed to ->
"In broad strokes, the journey of mastering data structures and algorithms can be divided into three stages:"